task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksDashboard.vue

```diff
@@ -183,6 +183,11 @@
           this.tasks = this.tasks.filter((t) => t.id !== event.detail);
         }
       });
+      this.$root.$on('update-cart', (event) => {
+        if (event) {
```

Comment:
You need to check wether the filter show completd is enabled or not ?

Reply:
this file is related to the TASKS project which works without filter. Did you mean the TasksViewDashboard file?

Reply:
No need to that. This file is related to the TASKS tab which works without filter. The filter is used in the PROJECTS tab and it was added to check if the task is completed before deleting it.

Reply:
> No need to that. This file is related to the TASKS tab which works without filter. The filter is used in the PROJECTS tab and it was added to check if the task is completed before deleting it.

I think you need to check if Show completed tasks is enabled or not, since if is enabled (the list content task completed and not completed) not need to do this filter.